src/style.css

```
@@ -1 +1,97 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  height: 60px;
+  padding: 0 50px;
+
+}
+
+.nav__list {
+  padding: 0;
+}
```

you have to add `display: flex` as well to have effect
```suggestion
.nav__list {
  display: flex;
  padding: 0;
  list-style: none;
}
```